README.md

```diff
@@ -64,9 +64,12 @@ Want to contribute to Mu?  Awesome!  Check out the [contributing guidelines](CON
 ## Building from source
 
 * Ensure [AWS CLI](http://docs.aws.amazon.com/cli/latest/userguide/cli-chap-getting-started.html) is configured with an access key, secret access key, and region.
-* Install Go tools 1.7+ - (https://golang.org/doc/install)
+* Install Go tools 1.9+ - (https://golang.org/doc/install)
+* Install [Go Dep](https://golang.github.io/dep/docs/installation.html)
 * Clone this repo `git clone git@github.com:stelligent/mu.git $GOPATH/src/github.com/stelligent/mu`
 * Go to src `cd $GOPATH/src/github.com/stelligent/mu`
+* Run `dep ensure` to install dependencies
```

Comment:
`make deps` also runs this.  just running `make` should run `make deps` and `make gen`

Reply:
Ok, maybe that's the missing piece. I already had go installed and cloned the repo then ran `dep ensure` and it failed because dep was not installed. I will go through the install again and try running make first. If that works then we should probably update the repo to include that step before mentioning running `dep ensure`